dockerfiles/Samtools/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM bashell/alpine-bash:latest
+
+ARG version=1.9
+
+RUN apk update \
+ && apk upgrade \
+ && apk add --no-cache \
+ build-base \
+ zlib-dev \
+ bzip2-dev \
+ xz-dev \
+ ncurses-dev \
+ ca-certificates \
+ wget
+
+RUN wget -q https://github.com/samtools/samtools/releases/download/${version}/samtools-${version}.tar.bz2
+
+RUN mkdir /samtools-${version}
+
+RUN tar -C / -xjvf samtools-${version}.tar.bz2
+
+WORKDIR /samtools-${version}/
+
+RUN make
+
+RUN mv /samtools-${version}/samtools /bin/ \
+ && rm -rf /samtools*
+
+RUN apk del build-base zlib-dev ca-certificates wget
```

Comment:
Unclear on need for this and subsequent command- space saving? Worth commenting

Reply:
Yes it is to reduce the size of the final image. I should be doing this more for all the dockerfiles but it takes time to figure out what can safely be removed once the program is built. Not sure about commenting as it is a fairly standard docker pattern - install, build, clean